tokens/testnet.json

```diff
@@ -19,15 +19,15 @@
     },
     {
       "id": "9a8cda8bc3423347b7fe2932ef7981ed67e8825dba3a5ee36011f578d1d70f00",
-      "name": "ETH (Portal)",
```

Comment:
`ETH (Portal)` is the convention we borrowed from Wormhole.

Reply:
yes, but then the metadata on chain doesn't match :/

Reply:
I see..